### PR TITLE
car admin list and update page

### PR DESCRIPTION
**Pull request by s3337852**

### CODE REVIEW

File: server/controllers/account/register.controller.js [hunk 1]
```diff
@@ -28,43 +28,38 @@ exports.register = function(req, res) {
   } else {
     let userLicense = new License();
     userLicense.licenseNumber = licensenum;
-    userLicense.disabled = false;
```
Comment (adrianchen91) [inline server/controllers/account/register.controller.js]: userLicense.disabled = false; removed but not set as isDisabled?

**Comment by s3337852** [comment]
License.isDisabled gets set to false by default 😉

File: server/controllers/account/register.controller.js (hunk 1 above)
Comment (s3337852) [reply server/controllers/account/register.controller.js]: License.isDisabled gets set to false by default 😉

**Comment by adrianchen91** [comment]
This is going to create a conflict with my pr lol

File: server/controllers/booking/booking.controller.js [hunk 2]
```diff
@@ -46,7 +46,6 @@ const createBooking = function(req, res) {
         newBooking.endsAt = endAt.toString();
         newBooking.user = selectedUser._id;
         newBooking.unlockCode = codeGenerator.generate();
-        newBooking.disabled = false;
```
[the remark this answers is not in the document]
Comment (s3337852) [reply server/controllers/booking/booking.controller.js]: yes in general that is correct, unless the field has a default value set
```
const bookingSchema = new Schema({
  unlockCode: { type: 'String', required: true, trim: true },
  startsAt: { type: 'Date', required: true },
  endsAt: { type: 'Date', required: true },
  lastAccessAt: { type: 'Date' },
  car: {
    type: mongoose.Schema.Types.ObjectId,
    ref: 'Car',
    required: true,
  },
  user: {
    type: mongoose.Schema.Types.ObjectId,
    ref: 'User',
    required: true,
  },
  damages: [{
    type: mongoose.Schema.Types.ObjectId,
    ref: 'Damage',
  }],
  isDisabled: { type: "Boolean", required: true, default: false },
  totalCost: { type: mongoose.Schema.Types.Number},
});
```

**Pull request merged**